### ISSUE TITLE
[BF] Export TruBudget Data to Excel

### ISSUE THREAD

**Issue by phpai** [openkfw/TruBudget#67]
In GitLab by @moise.djanhan on Jan 27, 2019, 22:41

Possibility to export Project, subproject and Workflows data to excel

**Comment by phpai** [comment]
In GitLab by @mathiashoeld on Jan 31, 2019, 15:01

Excel --> CSV
New endpoint to export TB data into CSV files based on user/group permissions

**Comment by phpai** [comment]
In GitLab by @kevinbader on Feb 25, 2019, 10:40

idea: export to csv as flat structure (recursively):

```csv
item id, subproject id, project id, donor1 budget, donor2 budget, project displayname, ...
```

@moise.djanhan there could be any number of donors for a given project - how should we deal with them?

with endpoint:

```
GET /project.csv
```

For items where the user doesn't have read permissions, the row should be redacted, i.e., filled up with null values. This should also be done on subproject and project level.

**Comment by phpai** [comment]
In GitLab by @moise.djanhan on Feb 26, 2019, 14:15

The user should be able to print only the project where he has the read permission. No matter if there are multiple donors.

**Comment by gonzochic** [comment]
Closed in #217
